.github/workflows/snyk.yml

```
@@ -0,0 +1,103 @@
+      - name: Snyk Code test
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        run: snyk code test --sarif-file-output=snyk-code.sarif || true
+
+      - name: Upload SARIF
+        if: always()
+        uses: github/codeql-action/upload-sarif@v3
+        with:
+          sarif_file: snyk-code.sarif
+
+
```

YAMLlint (1.37.1)

[error] too many blank lines (2 > 0)

(empty-lines)